CMakeLists.txt

```diff
@@ -27,12 +27,19 @@ include(options)
 include(cflags)
 set(${PROJECT_NAME_UPPER}_CXX_FLAGS "${LEATHERMAN_CXX_FLAGS}")
 add_definitions(${LEATHERMAN_DEFINITIONS})
-add_definitions(-DLEATHERMAN_I18N)
-## Pull in helper macros for working with leatherman libraries
-include(leatherman)
+
+if(LEATHERMAN_USE_LOCALES)
+    add_definitions(-DLEATHERMAN_I18N)
+    set(BOOST_COMPONENTS locale)
+endif()
+
+list(APPEND BOOST_COMPONENTS filesystem regex)
```

Comment:
Were we just using these accidentally without specifying them before?

Reply:
Yes! I hadn't even looked at again before yesterday...